src/app/card/info-status-card/examples/info-status-card-example.component.less

```diff
@@ -0,0 +1,24 @@
+@bottomMargin: 12px;
+.pfng-card-info-status {
+  @supports not (word-break: break-word) {
+    word-break: break-all;
+  }
+  @supports (word-break: break-word) {
+    word-break: break-word;
+  }
+}
+.cards-pf {
+  padding-bottom: 20px;
```

Comment:
with `View.Encapsulation: None;`, this may pose a problem with styles being injected into the `<head>` of the site.

Reply:
It's a good point, thanks @mindreeper2420 - I've prefixed the example selectors with a class which is specific to the info status card example, this should keep any conflict from arising.